### PR TITLE
FISH-6081: Fixing version for htmlunit

### PR DESCRIPTION
**Pull request by breakponchito**
Updating htmlunit version to fix CDI-TCK execution issues

### CODE REVIEW

**Review by aubi — CHANGES_REQUESTED** [review]
Please, add copyright years, then accepted.

File: glassfish-tck-runner/pom.template.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 
 -->
-
+<!-- Portions Copyright [2022] [Payara Foundation and/or its affiliates] -->
```
Comment (Pandrex247) [inline glassfish-tck-runner/pom.template.xml]: Payara is already above in the original blob, it just needs the year updating.
"Portions Copyright" is something specific to the dual license Payara uses I believe, not EPL.

**Comment by Pandrex247** [comment]
Does this PR have the correct target branch? Should this be going to the `master` branch or the `EE10` branch?

**Comment by breakponchito** [comment]
yes you are right the correct branch is the EE10, I already updated

File: glassfish-tck-runner/pom.template.xml [hunk 2]
```diff
@@ -420,7 +420,7 @@
                 <dependency>
                     <groupId>fish.payara.arquillian</groupId>
                     <artifactId>arquillian-payara-server-managed</artifactId>
-                    <version>3.0.alpha1</version>
+                    <version>3.0.alpha3</version>
```
Comment (Pandrex247) [inline glassfish-tck-runner/pom.template.xml:422]: Why not alpha5?

File: glassfish-tck-runner/pom.template.xml [hunk 3]
```diff
@@ -361,7 +361,7 @@
                 <dependency>
                     <groupId>fish.payara.arquillian</groupId>
                     <artifactId>arquillian-payara-server-remote</artifactId>
-                    <version>3.0.alpha1</version>
+                    <version>3.0.alpha3</version>
```
Comment (Pandrex247) [inline glassfish-tck-runner/pom.template.xml:363]: Why not alpha 5?

File: glassfish-tck-runner/pom.template.xml (hunk 2 above)
Comment (kalinchan) [reply glassfish-tck-runner/pom.template.xml:422]: There are issues with alpha5

File: glassfish-tck-runner/pom.template.xml (hunk 3 above)
Comment (Pandrex247) [reply glassfish-tck-runner/pom.template.xml:363]: Ah, because it causes the tests to freak out - interesting...

Comment (kalinchan) [reply glassfish-tck-runner/pom.template.xml:363]: This should be fixed in Patrik's PR https://github.com/payara/ecosystem-arquillian-connectors/pull/187 - it just needs to be merged and released

File: glassfish-tck-runner/pom.template.xml (hunk 1 above)
Comment (breakponchito) [reply glassfish-tck-runner/pom.template.xml]: already done

**Review by Pandrex247 — APPROVED** [review]

File: glassfish-tck-runner/pom.template.xml (hunk 3 above)
Comment (breakponchito) [reply glassfish-tck-runner/pom.template.xml:363]: The issue I saw when trying alpha 5 was that for some reason something is still maintaining the Servlet 5.0 Protocol and this connector version is using the Servlet 6.0. This is causing Nullpointer exception for most of the tests

**Comment by aubi** [comment]
Still missning license updates:
build.properties
glassfish-tck-runner/src/test/glassfish-managed/arquillian.template.xml

**Pull request merged**